File: web/src/model/victron_controller/dashboard/from_0_2_0_scheduled-action.ts
// @ts-nocheck
import {ScheduledAction as v0_2_0_ScheduledAction} from './v0_2_0/ScheduledAction'
import {ScheduledAction as dashboard_ScheduledAction} from './ScheduledAction'

export function convert__scheduled_action__from__0_2_0(from: v0_2_0_ScheduledAction): dashboard_ScheduledAction {
    return new dashboard_ScheduledAction (
        from.label,
        from.source,
        from.next_fire_epoch_ms,
        from.period_ms
    )
}

File: crates/dashboard-model/src/victron_controller/dashboard/from_0_2_0_scheduled_action.rs


pub fn convert__scheduled_action__from__0_2_0(from: &crate::victron_controller::dashboard::v0_2_0::scheduled_action::ScheduledAction) -> crate::victron_controller::dashboard::scheduled_action::ScheduledAction {
    crate::victron_controller::dashboard::scheduled_action::ScheduledAction {
        label: from.label.clone(),
        source: from.source.clone(),
        next_fire_epoch_ms: from.next_fire_epoch_ms.clone(),
        period_ms: from.period_ms.clone(),
    }
}